appstore/appstore/settings/base.py

```diff
@@ -339,6 +344,8 @@
     },
 }
 
+CSRF_TRUSTED_ORIGINS = os.environ.get("CSRF_DOMAINS", []).split(",")
```

Comment:
If `CSRF_DOMAINS` isn't defined in env then `([]).split` will error, should be `"[]"`

Reply:
haha I got it wrong too!